### PR TITLE
UXUI-808: HTTP calls

### PR DESCRIPTION
**Pull request by elliottcrush**
#### What does this PR do? (please provide any background)

Removes old http call to a bunch of assets. More information on the Jira

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><hr><h3>Application brand dependencies</h3><small><strong>A list of applications and whether or not they will automatically absorb your changes</strong></small></br><small>You're wanting to deploy <strong>2.2.5</strong></small><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/holidayextras/src/less/variables.less [hunk 1]
```diff
@@ -395,10 +395,9 @@
...
 @footer-bg:                       darken(@bg-secondary, 5%);
-    
+
 @footer-bottom-bg:                @bg-inverse;
 @footer-bottom-link:              rgba(255, 255, 255, 0.7);
-    
-@footer-logo:                     'http://open-source.holidayextras.com/ui-toolkit/images/holiday-extras-logo-white.png';
-@footer-logo-width:               80px;
 
+@footer-logo:                     '//open-source.holidayextras.com/ui-toolkit/images/holiday-extras-logo-white.png';
```
Comment (jodiedoubleday) [inline brands/holidayextras/src/less/variables.less]: this should  use the `asset-path` variable.

Comment (elliottcrush) [inline brands/holidayextras/src/less/variables.less]: Just pushed up again @jodiedoubleday - I've checked a few different applications to check they're still importing the font in the template. This appears to be the case but something we might need to be mindful of when merging - details of what applications might be affected is below.

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><hr><h3>Application brand dependencies</h3><small><strong>A list of applications and whether or not they will automatically absorb your changes</strong></small></br><small>You're wanting to deploy <strong>2.2.5</strong></small><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>2.1.6</strong> ❌</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.2</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.3</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><p>brandhx Version: <strong>^2.2.1</strong> ✅</p><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request closed without merging**